packages/compass-components/src/components/bson-value.tsx

```diff
@@ -96,11 +102,22 @@ export const ObjectIdValue: React.FunctionComponent<
 export const BinaryValue: React.FunctionComponent<
   PropsByValueType<'Binary'>
 > = ({ value }) => {
-  const { stringifiedValue, title } = useMemo(() => {
+  const { stringifiedValue, title, additionalHints } = useMemo(() => {
     if (value.sub_type === Binary.SUBTYPE_ENCRYPTED) {
       return {
         stringifiedValue: '*********',
         title: 'Encrypted',
+        additionalHints: (
+          <Link
+            className={encryptedHelpLinkStyle}
+            hideExternalIcon={true}
+            href="https://www.mongodb.com/docs/compass/current/in-use-encryption-tutorial/"
+            aria-label="Compass In-Use Encryption documentation"
+            title="Compass In-Use Encryption documentation"
```

Comment:
```suggestion
            title="Compass In-Use Encryption documentation"
            data-testid="bson-value-in-use-encryption-docs-link"
```
Not too useful of a test, so feel free to ignore, but we could test to be sure that this is added:
```ts
  it('should render an info link for encrypted values', function() {
    render(<BSONValue
      type="Binary"
      value={new Binary('encrypted data', Binary.SUBTYPE_ENCRYPTED)}
    />);

    expect(screen.findByTestId('bson-value-in-use-encryption-docs-link')).to.be.visible;
  })
```
With `screen` imported from `react-testing-library`.

Reply:
Thank you! :blue_heart: Done! :)